test/torchaudio_unittest/models/wav2vec2/fairseq_integration_test.py

```diff
@@ -207,6 +207,7 @@ def test_import_finetuning_model(self, config, _):
         imported = import_fairseq_model(original).eval()
 
         # Without mask
+        torch.manual_seed(0)
```

Comment:
Added line to fix failing test for CircleCI

Reply:
For the context of this PR, it looks good.